app/models/usersmodel.php

```diff
@@ -0,0 +1,16 @@
+<?php

```

Comment:
1. As convention, never use model prefix or postfix with model class names and file names.
2. Class names must always start with a capital letter